src/tests/ftest/osa/online_reintegration.py

```diff
@@ -60,8 +63,13 @@ def run_online_reintegration_test(self, num_pool, racer=False, server_boot=False
         pool = {}
         exclude_servers = (len(self.hostlist_servers) * 2) - 1
 
-        # Exclude one rank : other than rank 0.
-        rank = random.randint(1, exclude_servers)  # nosec
+        if multiple_ranks is True:
+            rank = "0,3"
+            # For multiple rank testing, we need RP_3G1 for IOR to complete.
+            oclass = "RP_3G1"
+        else:
+            # Exclude one rank : other than rank 0.
+            rank = random.randint(1, exclude_servers)  # nosec
```

Comment:
Similar here. It's better to use random so we are testing closer to a real-world scenario